finder-app/manual-linux.sh

```diff
@@ -47,34 +57,88 @@ then
     sudo rm  -rf ${OUTDIR}/rootfs
 fi
 
-# TODO: Create necessary base directories
+#Create the rootfs directory if it doesnt exist
+if [ ! -d "${OUTDIR}/rootfs" ]; then
+    mkdir -p "${OUTDIR}/rootfs"
+fi
+
+#Create the required directories
+# List of directories to create
+directories=("bin" "dev" "etc" "home" "conf" "lib" "lib64" "proc" "sbin" "sys" "lib/modules/${KERNEL_VERSION}" "tmp" "usr/bin" "usr/sbin" "var/log")
+
+# Loop through the directories and create them if they don't exist
+for dir in "${directories[@]}"; do
+    if [ ! -d "${OUTDIR}/rootfs/${dir}" ]; then
+        mkdir -p "${OUTDIR}/rootfs/${dir}"
+    fi
+done
 
+
+# COnfigure and compiliation of busybox
 cd "$OUTDIR"
 if [ ! -d "${OUTDIR}/busybox" ]
 then
 git clone git://busybox.net/busybox.git
     cd busybox
     git checkout ${BUSYBOX_VERSION}
-    # TODO:  Configure busybox
 else
     cd busybox
 fi
 
-# TODO: Make and install busybox
+make distclean
+make defconfig
```

Comment:
@Silex93 if you move this to the directory check above you could have faster incremental builds which don't need to rebuild busybox each time.